CMakeLists.txt

```diff
@@ -7,11 +7,19 @@ set(Python_ADDITIONAL_VERSIONS "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}"
 find_package(PythonLibs "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}" REQUIRED)
 find_package(TinyXML2 REQUIRED)
 
-catkin_package(
-  INCLUDE_DIRS include
-  LIBRARIES rospack ${PYTHON_LIBRARIES}
-  DEPENDS Boost TinyXML2
-)
+if(NOT APPLE)
+  catkin_package(
+    INCLUDE_DIRS include
+    LIBRARIES rospack ${PYTHON_LIBRARIES}
+    DEPENDS Boost TinyXML2
+  )
+else()
+  catkin_package(
+    INCLUDE_DIRS include
+    LIBRARIES rospack
+    DEPENDS Boost TinyXML2
+ )
+endif()
```

Comment:
To help reduce duplication, I think it would be nicer to do something like this:

```cmake
set(package_libraries rospack)
if(NOT APPLE)
  list(APPEND package_libraries ${PYTHON_LIBRARIES}
endif()

catkin_package(
  INCLUDE_DIRS include
  LIBRARIES ${package_libraries}
  DEPENDS Boost TinyXML2
)
```